### DISCUSSION TITLE
Show content of "Create Durables" in the overview

### DISCUSSION THREAD

**Discussion by ArminKugler** [postmanlabs/postman-flows#278] [Ideas]
Actually in my flow I see nicely the amoung of ms in delay and also the content of the condition block as well as the the content of assign variables. Only "Create Durables" does not reveal in the flow overview what Durables are set.
Since I am using this block a lot ( when working with asynchronous batch jobs you always get a batch job id in the first api call which you have to use for all following calls) would be nice to see that in this block I set the ID.

**Comment by samay-v** [comment]
Thanks for the feedback @ArminKugler. 
If I am not wrong you want a way a peek into what durables you've set from the block body itself (Instead of the modal).

**Comment by ArminKugler** [comment]
Hey semay, exactly. In the end my usecase is to make examples how to use my own API within the company and it would be cool to see that in this step of the flow the ID is set so people who are using this flow can at this place enter a different value and see what value is set there. Was just wondering why in the other types of flow steps the content is shown and here it is not.
